spec/integration/bisect_spec.rb

```diff
@@ -0,0 +1,158 @@
+RSpec::Support.require_rspec_core "formatters/bisect_progress_formatter"
+
+module RSpec::Core
+  RSpec.describe "Bisect", :slow, :simulate_shell_allowing_unquoted_ids do
+    include FormatterSupport
+
+    before do
+      skip "These specs do not consistently pass or fail on AppVeyor on Ruby 2.1+"
+    end if ENV['APPVEYOR'] && RUBY_VERSION.to_f > 2.0
+
+    def bisect(cli_args, expected_status=nil)
+      RSpec.configuration.output_stream = formatter_output
+      parser = Parser.new(cli_args + ["--bisect"])
+      expect(parser).to receive(:exit).with(expected_status) if expected_status
+
+      expect {
+        parser.parse
+      }.to avoid_outputting.to_stdout_from_any_process.and avoid_outputting.to_stderr_from_any_process
+
+      normalize_durations(formatter_output.string)
+    end
+
+    it 'finds the minimum rerun command and exits' do
```

Comment:
In respect to your last comment, this could be a cucumber scenario (as it'd be useful documentation of the output).